src/rpcTest/resources/privateNetConfig/conf/application.ini

```diff
@@ -1 +1 @@
--Dconfig.file=./conf/rpc-test-private.conf -Dlogback.configurationFile=./conf/rpc-test-logback.xml -J-Xss10M
+-Dconfig.file=./conf/rpc-test-private.conf -Dlogback.configurationFile=./conf/rpc-test-logback.xml -J-Xmx4g -J-Xss10M
```

Comment:
MAntis should be more than fine with 2g of memory with disabled mining. Do we really want to hardcode that value here and not on deployment side?